openwisp_controller/config/tests/__init__.py

```diff
@@ -18,5 +17,11 @@ class CreateConfigTemplateMixin(CreateTemplateMixin, CreateConfigMixin):
     def _create_config(self, **kwargs):
         if 'device' not in kwargs:
             kwargs['device'] = self._create_device(name='test-device',
-                                                   organization=kwargs.get('organization', None))
```

Comment:
I don't think this is right, why are you not honoring the `organization' that is passed and changing all the tests?

I think it would be better to do `organization=kwargs.pop('organization')` and avoid changing all the tests